src/lib.rs

```diff
@@ -56,13 +60,14 @@ impl Language for BoxedLanguage {
     fn too_high(&self) -> &'static str  { (**self).too_high() }
     fn ago(&self) -> &'static str       { (**self).ago() }
     fn place_ago_before(&self) -> bool  { (**self).place_ago_before() }
+    fn extra_space(&self) -> &str { (**self).extra_space() }
     fn get_word(&self, tu: TimeUnit, x: u64) -> &'static str  { 
         (**self).get_word(tu, x)
     }
 }
 
 /// Dynamic version of the `Language` trait
-pub type BoxedLanguage = Box<Language + Send + Sync + 'static>;
+pub type BoxedLanguage = Box<dyn Language + Send + Sync + 'static>;
```

Comment:
Are `Cargo.lock` and this changes significant? Won't it unnecessarily bump MSRV?